app/scripts/modules/core/src/pipeline/config/stages/managed/ImportDeliveryConfigExecutionDetails.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { get } from 'lodash';
+
+import { ExecutionDetailsSection, IExecutionDetailsSectionProps, StageFailureMessage } from 'core/pipeline';
+import { IGitTrigger } from 'core/domain';
+import { SETTINGS } from 'core/config';
+
+export function ImportDeliveryConfigExecutionDetails(props: IExecutionDetailsSectionProps) {
+  const { stage } = props;
+  const trigger = props.execution.trigger as IGitTrigger;
+  const errorDetailsAvailable = stage.isFailed && !stage.failureMessage && get(stage.context, 'error') != null;
```

Comment:
```suggestion
  const errorDetailsAvailable = stage.isFailed && !stage.failureMessage && stage.context.error;
```
I think you can skip the `get` wrapper here - we're accessing the `context` in a few other places just below, so it's either safe to access it directly here, or it's unsafe below! But I bet it's safe here.